tlspuffin/src/tls/fn_utils.rs

```diff
@@ -49,34 +53,61 @@ pub fn fn_append_transcript(
     Ok(new_transcript)
 }
 
-pub fn fn_decrypt_handshake(
-    application_data: &Message,
+pub fn fn_new_flight() -> Result<MessageFlight, FnError> {
+    Ok(MessageFlight::new())
+}
+
+pub fn fn_append_flight(flight: &MessageFlight, msg: &Message) -> Result<MessageFlight, FnError> {
+    let mut new_flight = flight.clone();
+    new_flight.messages.push(msg.clone());
+    Ok(new_flight)
+}
+
+pub fn fn_new_opaque_flight() -> Result<OpaqueMessageFlight, FnError> {
+    Ok(OpaqueMessageFlight::new())
+}
+
```

Comment:
For the record: it seems uncesseray to expose a fn_remove_nth() function symbol to remove an element from a flight since individual messages from a flights are also added as knowledges and the attacker can then always reconstruct a fligth with less elements. I suggest we document this somewhere.